### DISCUSSION TITLE
Deployment of new version breaking site for live users

### DISCUSSION THREAD

**Discussion by shiva-prasad-reddy** [nuxt/nuxt#16420] [Questions]
Is there any offical way to know that new updates are deployed and reload the current window. [NUXT 3]

Currenlty after new updates are deployed it showing the follwoing error messages:
`TypeError: Failed to fetch dynamically imported module: https://xyz.com/_nuxt/neww.3ce78252.js`

**Comment by shiva-prasad-reddy** [comment]
I found a way to reload the window, by attacher an error handler to vue router, but is this the right approach.
```
useRouter().onError((error, to, from) => {
  window.reload()
});
```

**Reply by TheAlexLichter** [reply]
Related: https://github.com/nuxt/framework/issues/6648

**Reply by shiva-prasad-reddy** [reply]
Thanks 🙇🏼
